doc/manual/src/release-notes/release-notes.md

```diff
@@ -5,8 +5,3 @@ Notable changes and additions are announced in the release notes for each versio
 
 Bugfixes can be backported on request to previous Nix releases.
 We typically backport only as far back as the Nix version used in the latest NixOS release, which is announced in the [NixOS release notes](https://nixos.org/manual/nixos/stable/release-notes.html#ch-release-notes).
-
-Backports never skip releases.
-If a feature is backported to version `x.y`, it must also be available in version `x.(y+1)`.
-This ensures that upgrading from an older version with backports is still safe and no backported functionality will go missing.
```

Comment:
If we end up keeping the current de facto situation that releases between the one used in NixOS and the latest won't have backported patches, we should at least state that explicitly here.

I still think it would be suboptimal, and things just working as one would expect would be way better UX than some random place in the manual saying "tough luck", but at least people will have a way to be sure about it.